pkg/kubelet/certificate/BUILD

```diff
@@ -0,0 +1,55 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Should this code actually be in k8s.io/client-go/util/cert? It's not kubelet specific.